static/js/src/certified-search-results.js

```diff
@@ -141,6 +142,13 @@ function toggleShowAllLinks() {
   });
 }
 
+function hideDrawerPageReload() {
+  const closeDrawerButton = document.querySelector(
+    ".p-side-navigation__toggle--in-drawer"
```

Comment:
I would use an `id` here? Would it be possible that another HTML item maybe around with the same class before this one?

Reply:
fixed